activities/scan_for_missing_chunks.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2020 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import json
+import pymysql
+import pymysql.cursors
+import time
+from boss_db import get_db_connection
+from bossutils import logger
+from ndingest.nddynamo.boss_tileindexdb import TASK_INDEX, MAX_TASK_ID_SUFFIX
+from ingestclient.core.backend import BossBackend
+
+"""
+Scans the tile index in DynamoDB for any chunks that have missing tiles.  If
+tiles are missing, they are placed back in the upload queue for that ingest
+job.  If there are missing tiles, the ingest job's state is reset to UPLOADING.
+"""
+
+log = logger.BossLogger().logger
+
+# Tile index attributes defined in ndingest.git/nddynamo/schemas/boss_tile_index.json.
+APPENDED_TASK_ID = 'appended_task_id'
+CHUNK_KEY = 'chunk_key'
+
+# This string is just hard coded in ndingest.git/nddynamo/boss_tileindexdb.py.
```

Comment:
Still less than ideal because it's still two places that it'll have to be updated.

Actually, I think I can import directly from the `ndingest` file and get rid of the code duplication. When I wrote this, I didn't think `ndingest` was on the activity server, but I see I'm using it in the clean up activity.